File: validacion-password.cs
using System;

class Program {
    static void Main() {
        // 1. Declarar variables (aquí hay que decir qué TIPO son: "string")
        string contrasenaSecreta = "Gatito123";
        string intentoUsuario = "";

        Console.WriteLine("--- SISTEMA DE SEGURIDAD ---");

        // 2. El Bucle WHILE (¡La lógica es idéntica!)
        while (intentoUsuario != contrasenaSecreta) {
            
            Console.Write("Ingrese la contraseña: ");
            
            // Así se pide el dato al usuario en C#
            intentoUsuario = Console.ReadLine();

            if (intentoUsuario != contrasenaSecreta) {
                Console.WriteLine("Acceso denegado. Intente de nuevo.");
            }
        }

        Console.WriteLine("¡Acceso Concedido! Bienvenido al sistema.");
    }
}

File: validacion_password.py
# Mi primer programa: Validar contraseña
# Autor: Diego Orellana
contrasena_secreta = "Ingeniero2026"
intento_usuario = ""

print("--- SISTEMA DE SEGURIDAD ---")

# Mientras la contraseña sea incorrecta, seguimos preguntando
while intento_usuario != contrasena_secreta:
    intento_usuario = input("Ingrese la contraseña: ")
    
    if intento_usuario != contrasena_secreta:
        print("Acceso denegado. Intente de nuevo.")

print("¡Acceso Concedido! Bienvenido al sistema.")
